CHANGES.md

```diff
@@ -2,6 +2,13 @@ Changes by Version
 ==================
 Release Notes.
 
+0.2.0
+------------------
+
+#### Bugs
+
+- Add volume claim for StatefulSet
```

Comment:
We have other changes for 0.2, right?

Reply:
No, we don't have others. Do you tend to move them to 0.1.1 ?

Reply:
Such as feature like adding support to cluster mode?

Reply:
I mean these commits, https://github.com/apache/skywalking-banyandb-helm/compare/v0.1.0...master

Reply:
done